app/assets/stylesheets/components/find/_call-out-box.scss

```diff
@@ -0,0 +1,16 @@
+.app-find-pink-callout {
+  border-left-color: #d43681;
+  background-color: #f3f2f1;
+  padding: 15px;
+  border-left-width: 5px;
+  min-height: 130px;
+
+  @include govuk-media-query($from: mobile) {
+    min-height: 105px;
+  }
```

Comment:
This seems content-specific. Is the min height here coupled to the content length?